.github/workflows/go.yml

```diff
@@ -0,0 +1,37 @@
+name: UnitTest CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    name: Build and Unit test
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.15
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Get dependencies
```

Comment:
I think you should use `go mod tidy` or `go mod download` here because you are using go modules for dependency management.